build.gradle

```diff
@@ -158,18 +177,17 @@ subprojects {
 
   artifacts {
     archives srcJar
+    archives testJar
+    archives testSrcJar
     archives javadocJar
   }
 
   plugins.withType(ScalaPlugin) {
-    //source jar should also contain scala source:
-    srcJar.from sourceSets.main.scala
-
     task scaladocJar(type:Jar) {
       classifier = 'scaladoc'
       from '../LICENSE'
       from '../NOTICE'
-      from scaladoc
+      from scaladoc.destinationDir
```

Comment:
Why this change? As far as I can tell it doesn't change anything and the previous version seems intuitive and regularly used in other projects.